### DISCUSSION TITLE
Sending Requests with Intercept?

### DISCUSSION THREAD

**Discussion by danreale** [cypress-io/cypress#9314] [Questions and Help] [answered]
I am looking to use the new intercept command with the news of server and route being deprecated. I am not sure if this is the right command to use going forward. This is my existing code... This is just an api call. Something that you would be able to send with postman.  Would I use the `intercept` command to send this request and verify responses, or would I still need to use `server` and `route`?  If `intercept` is the right function to use, could you provide an example? I didn't see anything in the examples for this case, unless it's documented in another way. What is the recommend way to perform api requests? I would need to use api requests for helpers within my tests.

```
it('POST ​/v1​/patients​/search', () => {
    cy.request({
      method: 'POST',
      url: '/api/patients/v1/patients/search',
      auth: {
        bearer: token
      },
      body: {
        firstName: utils.randomFirstName()
      }
    }).then((response) => {
      expect(response.status).to.eql(200);
      expect(response.body).to.have.property('results');
    });
  });
```

**Comment by bahmutov** [comment, marked as answer]
The `cy.request` command is NOT deprecated. You should use it to make your own API calls, like you are doing it. Additional reading: https://example.cypress.io/commands/network-requests#request and https://www.cypress.io/blog/2017/11/07/add-gui-to-your-e2e-api-tests/

The new `cy.intercept` command replaces `cy.server + cy.route` commands only

**Reply by danreale** [reply]
Ahh. Thanks @bahmutov. I guess I am in need of coffee this morning or glasses lol.
